scanner/mappers/mappers.go

```diff
@@ -615,16 +617,21 @@ func severityAndScores(vuln *claircore.Vulnerability, nvdScores map[string]nvdsc
 	default:
 		// Everything else uses NVD.
 		values.severity = vuln.Severity
```

Comment:
> and we will need to use the CVSS score to determine severity (for example Alpine)

I am not sure we want to default to NVD for things customers might want to make decisions upon. If the security source is returning empty severity for Alpine, and we show NVD without letting them know we are using NVD, it can cause confusion.

We don't use `Severity` in Central, so empty just means no scores, and we skip.

Regarding the cases you linked, I think it should be handled by the parser; I don't want to default to NVD unnecessarily and without customers knowing it.

Reply:
(BTW, Description is fine, because it's cosmetic and to some extend harmless. But it can be that Description suffers from the same potential problem.)

Reply:
> Regarding the cases you linked, I think it should be handled by the parser; I don't want to default to NVD unnecessarily and without customers knowing it.

Central-side or Scanner mapper? Does our mapper handle this, or do we need a ticket?

Reply:
> Central-side or Scanner mapper?

The parser in ClairCore. Not by us using NVD to fill in.